readme.md

```diff
@@ -32,6 +32,12 @@ console.log(results);
 */
 ```
 
+In ES6/Typescript the first line should look like this instead:
+
+```js
+import unquotedValidator = require('unquoted-property-validator');
```

Comment:
Should be from no?

Reply:
I couldn't get it to work using the "normal" syntaxes (inluding from) so I used the solution suggested here and it worked: https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this#39415662

Before I added the proposed typescript bindings in my local project I was avble to use it like this:
```es
import * as unquotedValidator from 'unquoted-property-validator';
```